packages/video_player/video_player_avfoundation/example/macos/Runner/AppDelegate.swift

```diff
@@ -5,7 +5,7 @@
 import Cocoa
 import FlutterMacOS
 
-@NSApplicationMain
+@main
```

Comment:
`@NSApplicationMain` was deprecated, tool migrated it: https://github.com/flutter/flutter/pull/146848